controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go

```diff
@@ -63,7 +65,18 @@ func (r *ClusterReconciler) ReconcileAddonSecret(
 		return res, err
 	}
 	secret = newAddonSecret.DeepCopy()
-	return res, r.Update(ctx, secret)
+	if err := r.Update(ctx, secret); err != nil {
+		log.Error(err, "Failed to update ako add on secret, requeue")
+		return res, err
+	}
+
+	// patch cluster bootstrap here
+	if err := r.patchAkoPackageRefToClusterBootstrap(ctx, cluster); err != nil {
```

Comment:
i know its from previous design discussion, but why can't we deploy a TKR where the ClusterBoostrapTemplate has ako as additionalpackage?